src/firebolt/async_db/cursor.py

```diff
@@ -245,21 +312,9 @@ async def execute(
         set_parameters: Optional[Dict] = None,
     ) -> int:
         """Prepare and execute a database query. Return row count."""
-        start_time = time.time()
 
-        # our CREATE EXTERNAL TABLE queries currently require credentials,
-        # so we will skip logging those queries.
-        # https://docs.firebolt.io/sql-reference/commands/ddl-commands#create-external-table
-        if not re.search("aws_key_id|credentials", query, flags=re.IGNORECASE):
-            logger.debug(f"Running query: {query}")
-
-        self._reset()
-        resp = await self._do_execute_request(query, parameters, set_parameters)
-        self._store_query_data(resp)
-        self._state = CursorState.DONE
-        logger.info(
-            f"Query fetched {self.rowcount} rows in {time.time() - start_time} seconds"
-        )
+        params_list = [parameters] if parameters else []
```

Comment:
I'm not entirely understanding this line. As I understand the user would call `execute("INSERT ... VALUES (?, ?)", [(1,2), (3,4), (5,6)])` is that not what we want to pass to the _do_execute_request?

Reply:
No, users can only pass a list of parameters to executemany.
In case of execute, only a single set of parameters is passed.
So you can only do `execute("INSERT ... VALUE (?, ?)", (1, 2))`

Reply:
From PEP249
execute: `Parameters may be provided as sequence or mapping and will be bound to variables in the operation`
executemany: `Prepare a database operation (query or command) and then execute it against all parameter sequences or mappings found in the sequence seq_of_parameters.`

Reply:
Makes sense